pkg/token/token.go

```diff
@@ -78,6 +78,9 @@ type Identity struct {
 	// in conjunction with CloudTrail to determine the identity of the individual
 	// if the individual assumed an IAM role before making the request.
 	AccessKeyID string
+
+	// ASW STS endpoint type(global/regional) used to authenticate (expected values sts_global/sts_regional)
+	STSEndpointType string
```

Comment:
Rather than a type with "global" or "regional", lets put the region the request was signed to here. It'll be more informative

Reply:
updated code to log endpoint instead of global/regional